plugins/traefik-vhosts/templates/compose.yml.sigil

```diff
@@ -0,0 +1,58 @@
+---
+version: "3.3"
+
+services:
+  traefik:
+    image: "{{ $.TRAEFIK_IMAGE }}"
+    command:
+      - --entrypoints.http.address=:80
+      {{ if $.TRAEFIK_LETSENCRYPT_EMAIL }}
+      - --entrypoints.https.address=:443
+      {{ end }}
+      - --providers.docker
+      - --api={{ $.TRAEFIK_API_ENABLED }}
+      - --api.dashboard={{ $.TRAEFIK_DASHBOARD_ENABLED }}
+
+      - --log.level={{ $.TRAEFIK_LOG_LEVEL }}
+      - --log.format=json
+
+      {{ if $.TRAEFIK_LETSENCRYPT_EMAIL }}
+      - --certificatesresolvers.leresolver.acme.caserver=https://acme-staging-v02.api.letsencrypt.org/directory
+      - --certificatesresolvers.leresolver.acme.email={{ $.TRAEFIK_LETSENCRYPT_EMAIL }}
+      - --certificatesresolvers.leresolver.acme.storage=/acme.json
+      - --certificatesresolvers.leresolver.acme.tlschallenge=true
+      {{ end }}
+
+    labels:
+      # Dashboard
+      - "traefik.http.routers.api.rule=Host(`{{ $.TRAEFIK_API_VHOST }}`)"
+      - "traefik.http.routers.api.service=api@internal"
+      - "traefik.http.routers.api.entrypoints={{ if $.TRAEFIK_LETSENCRYPT_EMAIL }}https{{ else }}http{{ end }}"
+      - "traefik.http.routers.middlewares=auth"
```

Comment:
What version of Traefik was used to test this basic_auth?

Shouldn't this middleware be under `.api`? i.e.
```diff
-     - "traefik.http.routers.middlewares=auth"
+     - "traefik.http.routers.api.middlewares=auth"
```

Reply:
traefik:v2.8